unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldController.cs

```diff
@@ -485,6 +485,10 @@ public void TryStartEnterEditMode(bool activateCamera, IParcelScene targetScene
             return;
         }
 
+        //If the scene is still not loaded, we return as we still can't enter in builder in world 
+        if (sceneToEditId != null)
+            return;
```

Comment:
Only by curiossity: Why it is necessary to do this check by second time? As I can see, the `FindSceneToEdit()` function is not any kind of coroutine nor an async method... in what case could we get the `sceneToEditId` as null in this point?

Reply:
In the case that we try to enter in the land but the scene is still not loaded, so when we try to find the scene we can't find it since it is not loaded yet, To do this you need to enter with the 'K' key very quickly after we exit.